docs/CLIENT.md

```diff
@@ -449,6 +455,11 @@ Following error types are exposed:
   * `InternalServerError`
   * `ServiceUnavailable`
 
+#### Correlation Header ID
+When a request fails with `statusCode >= 400` the reponse contains a special header `x-correlation-id`. This is specially useful for the SPHERE.IO team in order to better correlate failure requests with other possible causes.
```

Comment:
Ah ok. And I actually saw that headers were already returned...will adjust a couple of things then